app/_kong_plugins/hmac-auth/examples/enforce-headers-body-validation-algorithms.yaml

```diff
@@ -8,6 +8,13 @@ requirements:
  - You have created a [Consumer with HMAC credentials](/plugins/hmac-auth/#consumer-authentication)
 
 
+values:
+  enforce_headers:
+    description: List of headers that must be signed
+  algorithms:
+    description: List of acceptable HMAC algorithms for the digest
+  validate_request_body:
+    description: Enable or disable request body validation
```

Comment:
This doesn't do anything, unless you're using this example to suggest something we should add to the platform.

```suggestion
```